src/components/HomeSection.jsx

```diff
@@ -5,61 +5,62 @@ import { Container, Grid, Typography } from '@mui/material';
 const HomeSection = ({
   alignItems,
   direction,
-  tab,
   title,
+  main,
   description,
   buttons,
   pic,
-}) => (
-  <Container sx={{ padding: 4 }}>
-    <Grid container spacing={2} direction={direction} alignItems={alignItems}>
-      <Grid item xs={12} sm={6}>
-        <Typography variant="overline" component="h6" gutterBottom>
-          {tab}
-        </Typography>
-        <Typography variant="h2" component="h2" gutterBottom>
-          {title}
-        </Typography>
-        <Typography variant="body2" paragraph>
-          {description}
-        </Typography>
-        <Grid container spacing={2}>
-          {buttons.map((button, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Grid item key={i}>
-              {button}
-            </Grid>
-          ))}
+}) => {
+  const picSize = main ? '90%' : '80%';
+  return (
+    <Container sx={{ padding: 4 }}>
+      <Grid container spacing={3} direction={direction} alignItems={alignItems}>
+        <Grid item xs={12} sm={6}>
+          <Typography variant={main ? 'h2' : 'h3'} component="h2" gutterBottom>
+            {title}
+          </Typography>
+          <Typography variant="body2" paragraph>
+            {description}
+          </Typography>
+          <Grid container spacing={2}>
+            {buttons.map((button, i) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <Grid item key={i}>
+                {button}
+              </Grid>
+            ))}
+          </Grid>
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <img
+            src={pic}
+            alt=""
+            style={{
+              width: picSize,
+              height: picSize,
+              objectFit: 'contain',
+            }}
+          />
         </Grid>
       </Grid>
-      <Grid item xs={12} sm={6}>
-        <img
-          src={pic}
-          alt=""
-          style={{
-            width: '100%',
-            height: '100%',
-            objectFit: 'contain',
-          }}
-        />
-      </Grid>
-    </Grid>
-  </Container>
-);
+    </Container>
+  );
+};
 
 HomeSection.defaultProps = {
   alignItems: 'flex-start',
+  main: false,
   direction: 'row',
-  tab: '',
   buttons: [],
+  description: undefined,
 };
 
 HomeSection.propTypes = {
   alignItems: PropTypes.string,
   direction: PropTypes.string,
-  tab: PropTypes.string,
   title: PropTypes.string.isRequired,
-  description: PropTypes.string.isRequired,
```

Comment:
Shouldn't this be required instead of undefined?

Reply:
The last part of the page also uses this component and doesn't have any description text.